src/clj/witan/send/send.clj

```diff
@@ -13,3 +13,9 @@
    (-> (i/build-input-datasets (:project-dir config) (:file-inputs config) (:schema-inputs config))
        (p/prepare-send-inputs (:transition-parameters config))
        (r/run-send-model (:projection-parameters config)))))
+
+(defn input-analysis
```

Comment:
It's just a helper function, not even used for actual CSV ouputs.  I left it in as I thought it might be useful for others.  The result is the input map with all the different distribution parameters.

Reply:
👍